cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectS2ICluster.java

```diff
@@ -28,13 +28,16 @@
 import io.strimzi.api.kafka.model.KafkaConnectS2IAssembly;
 import io.strimzi.api.kafka.model.KafkaConnectS2IAssemblySpec;
 
+import java.util.Map;
+
 public class KafkaConnectS2ICluster extends KafkaConnectCluster {
 
     // Kafka Connect S2I configuration
     protected String sourceImageBaseName = DEFAULT_IMAGE.substring(0, DEFAULT_IMAGE.lastIndexOf(":"));
     protected String sourceImageTag = DEFAULT_IMAGE.substring(DEFAULT_IMAGE.lastIndexOf(":") + 1);
     protected String tag = "latest";
     protected boolean insecureSourceRepository = false;
+    static Map<String, String> configMapS2I;
```

Comment:
Why is this `static`?

Reply:
This map is used to save "previous" configuration which is later compared to the current configuration.